apps/keyboard/js/layouts/emoji.js

```diff
@@ -0,0 +1,458 @@
+Keyboards.emoji = {
+  label: 'Emoji',
+  types: ['text', 'url', 'email'],
+  menuLabel: 'Emoji',
+  pages: [{   // Page 0 - People
+    pageType: 'emoji',
+    sectionKeys: [ { compositeKey: '😄', type: 'emoji' }, { compositeKey: '😃', type: 'emoji' },
```

Comment:
You obvious have problem understand

https://github.com/mozilla-b2g/gaia/pull/27742#discussion_r24080313

Any reason why we can't specify the sections manually and instead have to use for loop to split the keys into sections?